### PR TITLE
questStringRaw translation for RDM systems with WatWowMap/pogo-translations

### PR DESCRIPTION
**Pull request by ReuschelCGN**
**needs testing but works with german translations**

could be switched on in config with "useRDMQuestString": true, and didnt changed the old way
fetch all locales from: 
`https://raw.githubusercontent.com/WatWowMap/pogo-translations/master/static/locales`

NO support for MAD sorry!

### CODE REVIEW

**Comment by Fabio1988** [comment]
I would change config to `useQuestTitleString`. But as many don't know what's that about, I would call it `useIngameQuestString`.

Default to true :)

File: src/controllers/quest.js [hunk 1]
```diff
@@ -248,10 +253,25 @@ class Quest extends Controller {
 
 				const language = cares.language || this.config.general.locale
 				const translator = this.translatorFactory.Translator(language)
+				const questI18l = require(path.join(__dirname, `../util/locale/i18n_${language}.json`))
```
Comment (Fabio1988) [inline src/controllers/quest.js:256]: `questI18n` 👍

File: config/default.json [hunk 2]
```diff
@@ -72,6 +72,8 @@
      "defaultTemplateName": 1,
     // library of custom DTS templates
       "dtsDictionary": {},
+    // Queststring i18l translation for RDM systems
```
[the remark this answers is not in the document]
Comment (ReuschelCGN) [reply config/default.json]: changed

File: src/controllers/quest.js (hunk 1 above)
Comment (ReuschelCGN) [reply src/controllers/quest.js]: changed

**Pull request closed without merging**